src/activities/CreateDirectLineMeasurement3D.ts

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import type {
+    IActivityHandler,
+    IActivityContext,
+} from "@geocortex/workflow/runtime/IActivityHandler";
+import { MapProvider } from "@geocortex/workflow/runtime/activities/arcgis/MapProvider";
+import { activate } from "@geocortex/workflow/runtime/Hooks";
+import DirectLineMeasurement3D from "@arcgis/core/widgets/DirectLineMeasurement3D";
+import SceneView from "@arcgis/core/views/SceneView";
+type removeFunction = (() => void);
```

Comment:
These two types are leftover and can be deleted.

Reply:
done